browser/resources/settings/default_brave_shields_page/default_brave_shields_page.html

```diff
@@ -48,9 +51,7 @@
         </div>
       </div>
     </div>
-    <div class="settings-box">
-      <div class="label shields-primary-title">$i18n{shieldsLookFeelTitle}</div>
-    </div>
+    <hr />
     <settings-toggle-button
         class="continuation"
```

Comment:
It would be better to use `class="cr-row"` instead of using `<hr />` as `<hr />` gives more thicker line than others.